openapi/index.yaml

```diff
@@ -338,25 +338,181 @@ paths:
         Note that this is an optimization for those services that need to send very high
         volumes of messages per day, to avoid having to make two API calls for each message.
       responses:
-        '200':
+        "200":
           description: Found.
           schema:
             $ref: "#/definitions/SubscriptionsFeed"
           examples:
             application/json:
               dateUTC: "2019-09-03"
-              subscriptions: [ "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855" ]
-              unsubscriptions: [ ]
-        '401':
+              subscriptions:
+                [
+                  "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+                ]
+              unsubscriptions: []
+        "401":
           description: Unauthorized
-        '404':
+        "404":
           description: Subscriptions feed not available yet.
           schema:
             $ref: "#/definitions/ProblemJson"
-        '429':
+        "429":
           description: Too many requests
       parameters:
-      - $ref: "#/parameters/DateUTC"
+        - $ref: "#/parameters/DateUTC"
+  "/services":
+    post:
+      operationId: createService
+      summary: Create Service
+      description: |
+        Create a new Service with the attributes provided in the request payload.
+      parameters:
+        - in: body
+          name: body
+          description: The service payload.
+          schema:
+            "$ref": "#/definitions/ServicePayload"
+      responses:
+        "200":
+          description: Service created.
+          schema:
+            "$ref": "#/definitions/ServiceWithSubscriptionKeys"
+        "400":
+          description: Invalid payload.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+        "401":
+          description: Unauthorized.
+        "429":
+          description: Too many requests.
+        "500":
+          description: Internal Server Error.
+    get:
+      operationId: getServices
+      summary: Get Services
+      description: |
+        Retrieve a list of services.
```

Comment:
Since this endpoint will map to GetUser (and not to getServices) I'd rather rename summary and operation as
"Get User Services: retrives the id of the services tied to the calling user".

Reply:
Fixed in [d1fea39](https://github.com/pagopa/io-functions-services/pull/67/commits/d1fea39496f18158848e087c920b5e5e6c3c0073)